torchani/cuaev/aev.cu

```diff
@@ -26,6 +26,52 @@ constexpr int csubaev_offsets(int i, int j, int n) {
   return starting + offset;
 }
 
+#define MAX_NSPECIES 10
+#define RCR_X 0
+#define RCA_X 1
+#define RADIAL_SUBLENGTH_X 2
+#define RADIAL_LENGTH_X 3
+#define ANGULAR_SUBLENGTH_X 4
+#define ANGULAR_LENGTH_X 5
+#define NUM_SPECIES_X 6
+#define ETAR_X 7
+#define ETAA_X 8
+#define ZETA_X 9
+#define SUBAEV_OFFSET_X 10
+
+__constant__ float AEV_CONSTANTS[SUBAEV_OFFSET_X + MAX_NSPECIES * MAX_NSPECIES];
```

Comment:
Instead of using globals, I think constants should be a member of `CuaevComputer`. Using globals only allows you to have one instance of aev computer to work.